transport/src/androidMain/kotlin/com/genesys/cloud/messenger/transport/util/logs/OkHttpLogger.kt

```diff
@@ -0,0 +1,6 @@
+package com.genesys.cloud.messenger.transport.util.logs
+
+import okhttp3.logging.HttpLoggingInterceptor
+
+internal fun Log.okHttpLogger(): HttpLoggingInterceptor.Logger =
+        HttpLoggingInterceptor.Logger { message -> kermit.i(message) }
```

Comment:
The linter is complaining about spacing here

> /Users/jenkins/opt/jenkins-swarm-client/workspace/enger-transport-mobile-sdk_PR-88/transport/src/androidMain/kotlin/com/genesys/cloud/messenger/transport/util/logs/OkHttpLogger.kt:6:1: Lint error > [indent] Unexpected indentation (8) (should be 4)

Reply:
Fixed. I thought I'd run the linter against it. I _had_... then ignored the error. 😬